PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -382,8 +382,8 @@ def _add_deepFlavour(process):
         toKeep = ['deepTau2017v2p1','deepTau2018v2p5']
     )
     from Configuration.Eras.Modifier_phase2_common_cff import phase2_common #Phase2 Tau MVA
-    phase2_common.toModify(tauIdEmbedder.toKeep, func=lambda t:t.append('newDMPhase2v1')) #Phase2 Tau isolation MVA
-    phase2_common.toModify(tauIdEmbedder.toKeep, func=lambda t:t.append('againstElePhase2v1')) #Phase2 Tau anti-e MVA
+    _tauIds_phase2 = ['newDMPhase2v1','againstElePhase2v1']
```

Comment:
@mbluj `newDMPhase2v1` was not kept in the master version. Just to try to understand: why?

Reply:
> @mbluj `newDMPhase2v1` was not kept in the master version. Just to try to understand: why?

It has quite likely something to do with https://github.com/cms-sw/cmssw/pull/40724#issuecomment-1424301473, but I fail to find the connection (sorry...)